scipy/stats/tests/test_stats.py

```diff
@@ -7917,6 +7917,222 @@ def test_alias(self):
         assert_equal(res.stat, res.statistic)
 
 
+class TestQuantileTest():
+    r""" Test the non-parametric quantile test, 
+    including the computation of confidence intervals
+    """
+    
+    np.random.seed(0)
+    m = 100                                     # number of tests
+    N = np.random.randint(10, 100, size=m)      # num of samples
+    x = []                                      # sorted samples
+    for i in range(m):
+        num = N[i]
+        x.append(np.sort(np.random.permutation(1000)[:num]))
+    quantile_def = np.random.rand(m)            # quantile definition in [0,1)
+    quantile_val = np.random.rand(m)*100        # quantile value in [0,100)
+    confidence = 0.25*np.random.rand(m)+0.75    # confidence in [0.75,1)
+
+    def test_quantile_test_iv(self):
+        
+        x = [1,2,3]
```

Comment:
```suggestion
        x = [1, 2, 3]
```
Spaces after commas here and below. [Demonstrated by example](https://stackoverflow.com/questions/31741418/pep8-space-after-a-comma) in PEP8 standard, and mostly adhered to in modern SciPy code.

Reply:
ack, I'll change that